### PR TITLE
Create awesome-books-spa branch

### PR DESCRIPTION
**Pull request by koomekevo**
Add single page application (spa) functionality.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -29,6 +29,10 @@ class Library {
 }
```
Comment (SajjadAhmad14) [inline index.js:35]: [ OPTIONAL BUT RECOMMENDED ] After adding the books to the list when the user refreshes the page list becomes empty. Please make sure that your data is preserved in the browser and do not get lost on refresh.

File: index.html [hunk 2]
```diff
@@ -16,25 +16,54 @@
 <body>
```
Comment (SajjadAhmad14) [inline index.html:16]: Everything is according to the wireframe! But please do not disable eslint rules and resolve the issues instead.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Kevin and Nat, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.js (hunk 1 above)
Comment (koomekevo) [reply index.js:35]: We have tried to implement this with some level of success

File: index.html (hunk 2 above)
Comment (koomekevo) [reply index.html:16]: luxon.js has hundreds of linter errors so we disabled it.

**Review by mke2111 — APPROVED** [review]
### Hi @koomekevo & @NataliaPoletaeva, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**